Assignment4.java

```diff
@@ -0,0 +1,87 @@
+package assignment4;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class Assignment4 {
+
+    @Test
+    public void testNoIslands() {
+        boolean[][] grid = new boolean[][] {
+                {false, false, false},
+                {false, false, false}
+        };
+        int result = new IslandCounter(grid).countIslands();
+        assertEquals("Found some islands", 0, result);

```

Comment:
In your messages in these tests, they don't tell you much more than assertEquals() would without a message. So you could consider writing without the message, it doesn't gain much